include/nuttx/dma/dma.h

```diff
@@ -246,10 +251,13 @@ struct dma_config_s
   unsigned int direction;
   unsigned int priority;
   unsigned int timeout;
+  unsigned int priv;
```

Comment:
Should this be `uintptr_t` or `unsigned int` is enough?

Reply:
`unsigned int`  will be ok